Dynavity/Dynavity/persistence/storage/CloudStorageManager.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import FirebaseDatabase
+import CodableFirebase
+import Combine
+
+struct CloudStorageManager {
+    let encoder = FirebaseEncoder()
+    let decoder = FirebaseDecoder()
+    let userId = OnlineCanvas.myUserId
```

Comment:
I've changed it to go through `OnlineCanvasDTO` instead of querying `OnlineCanvas` directly, since we are dealing with `OnlineCanvasDTO` more. That should help?